be/src/util/disk_info.cpp

```diff
@@ -64,9 +62,6 @@ void DiskInfo::get_device_names() {
             continue;
         }
 
-        // Remove the partition# from the name.  e.g. sda2 --> sda
```

Comment:
will it cause problem if is a hdd driver named as `sdxx##` or virtual disk like `vd[abc][0-9]`?

Reply:
Yes, I have add some details in the PR.